bucket/jcpicker.json

```diff
@@ -0,0 +1,36 @@
+{
+    "homepage": "https://annystudio.com/software/colorpicker/",
+    "description": "Free portable offline colour picker and colour editor for web designers and digital artists.",
+    "version": "5.5",
+    "license": "Freeware",
+    "architecture": {
+        "64bit": {
+            "url": "https://annystudio.com/jcpicker.exe",
+            "hash": "530047d43a009458d56f326b1e2d53cde19b79fe4c49def0f98ddd8e90190b34"
+        },
+        "32bit": {
+            "url": "https://annystudio.com/jcpicker_32bit.exe#/jcpicker.exe",
+            "hash": "0762c6718020b804a5e7286bdc9a53fc772eb3a5f17e2d1ede2584a964bf8242"
+        }
+    },
+    "bin": "jcpicker.exe",
```

Comment:
Available parameters?

Reply:
No parameters. Just the command `jcpicker` will start the GUI picking tool.

Reply:
Then completely useless

Reply:
Why? 
For comparison, instant-eyedropper (in extras bucket) cannot be invoked from commandline, you always need to press its hotkey. (running instant-eyedropper from commandline just starts its background process, it doesn't bring up the picker)

Reply:
Not a single reason to flood users shims path with completely useless shim, which has no command line support.

Reply:
How else do you start a GUI application from the commandline? You need the shim, right?

Reply:
Starting gui from command line is useless from the initial, when you cannot control how it will be started or what will be shown (vscode, npp, explorer).

Reply:
Hmm, I understand your reason that since it supports no parameters, it's the same as launching a short cut.

But being on the commandline saves time (otherwise you go to start menu, use your mouse to click and launch apps).

Feel free to close this PR if you feel it won't save time/convenience.